iroha/src/account.rs

```diff
@@ -136,14 +136,25 @@ pub mod isi {
         RemoveSignatory(<Account as Identifiable>::Id, PublicKey),
     }
 
+    /// Enumeration of all legal Outputs for `AccountInstruction` execution.
+    #[derive(Debug)]
+    pub enum Output {
+        /// Variant of output for `AccountInstruction::TransferAsset`.
+        TransferAsset(WorldStateView),
```

Comment:
Do we need the whole world state view as an output for all these instructions? For example for transfer, just the two accounts and their data as an output would be sufficient to my mind. Or is it only a draft and will be corrected later?

Reply:
It's not the draft, yet not a final solution. Maybe you remember our discussion about ISI execution optimizations with diffs. 
We should do that, but it's not a small change and I don't want to do it in the same PR. This PR will give rollbacks by the same mechanism that was used before.

Reply:
Ok, sure, I also think it should be in a different PR.